index.html

```diff
@@ -12,24 +12,30 @@
 </head>
 <body>
   <div class="status-bar">
-    <div class="hour">9:41</div>
-    <div class="wifi"></div>
-    <div class="signal"></div>
-    <div class="battery"></div>
+    <div class="status-hour">9:41</div>
+    <div class="status-bar-space"></div>
+    <div class="status-images">
+      <img src="./Images/status.png">
+    </div>    
   </div>
-  <nav class="navbar">
-     
+  <nav class="navbar">     
     <div class="nav-name">Jhon Doe</div>
-    <div class="nav-menu"></div>      
-        
+    <div class="nav-space"></div>
+    <div class="nav-menu"><img src="./Images/menu.png" height="30px"></div>        
   </nav>
   <div class="container">
     <div class="primary-text">Hey There. I’m Alex</div>
     <div class="supporting-text">I’am a Software Developer</div>
     <div class="description">I can help you build a product , feature or website. 
       Look through some of my work and experience! If you like what you see and have a project you need coded, 
       don’t hesitate to contact me.</div>
-  </div>
-    
+    <div class="icons">
+      <img src="./Images/Icon -GitHub.png">
+      <img src="./Images/Icon - Linkedin.png">
+      <img src="./Images/Icon - Angelist.png">
+      <img src="./Images/Icon - Twitter.png">
+      <img src="./Images/Icon - Medium.png">     
+    </div>
+  </div>   
```

Comment:
- [ ] Great job @dfsalam  you have very clean markup :+1: However, according to the [project's requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), this section should not be inside a `<div>` tag but a `<section>` tag. Kindly change the `<div>` to `<section>` an this should be good.